packages/v4/src/content/developer-resources/accessibility-development.md

```diff
@@ -98,3 +98,24 @@ The recommended interaction pattern for the modal components like the modal or p
 - For screen reader users, the other contents on the page should be hidden from the screen reader.
 
 Based on the [screen reader / browser combinations](https://www.patternfly.org/v4/get-started/accessibility-guide#testing) we use for testing, we recommend applying `aria-hidden="true"` to the parent wrapping element of the page contents. Note that the modal element of the component must not be a descendent of this element with `aria-hidden="true"` and should be included as a sibling to this element.
+
+### Scrollable elements
+Any elements with horizontal or vertical scroll need to be accessible via keyboard. It may be necessary to ensure every 
+container with an overflow rule and content which could possibly grow larger than its parent's height to have 
+`tabindex=0`. This will ensure that parent can capture browser focus and be scrolled using the arrow keys. Some browsers
+will automatically make any scrollable element focusable to ensure keyboard accessibility as long as they have a 
+non-negative tabindex.
+
+Additionally, when adding tabIndex in this scenario, it would also be beneficial to add an aria-label that describes the scrollable element.
+Some screen reader users navigate page elements via an element list or rotor menu and the default label for these scrollable elements
+will be driven by whatever text content exists inside the container. This generated label may not be suitable to describe the regions purpose, and 
+would likely be better described with a simple aria-label.
```

Comment:
Maybe when we open and begin tackling an issue to address the use of aria-labels on divs or other places where that issue is flagged, we can also update these docs. As far as I can tell, axe is flagging it as 'needs review' and on the wizard examples, it make sense that - because of the rotor menu navigation as spaxman noted - these situational uses passes review (for me). WDYT?

Reply:
That's a fair point. I suppose between aXe only telling us it isn't "well supported" and Jessie not having had any issues/not finding anything official (I also haven't really had much issues other than some potential JAWS behavior, but that would require more specific testing), it could be worth waiting to see if we hear about any issues/come across any resources about it down the line.

If we were to add anything now, something along the lines of "This may not be well supported in some circumstances, but because of X, Y, and Z we are opting to do this" might be better, but I'd be okay holding off on adding any verbiage about it for now. Worse case is one of us tests or comes across something a few days from now and update the docs then.

Reply:
I actually did a little more reading and found this blog: https://www.tpgi.com/short-note-on-improving-usability-of-scrollable-regions/

it lead me to read up on use of `role='region'` and using that role seems to remove the warnings. What do you think? 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role